### PR TITLE
Add default k6 user-agent string

### PR DESCRIPTION
**Pull request by robingustafsson**
Implementation of https://github.com/loadimpact/k6/issues/310.

### CODE REVIEW

**Comment by liclac** [comment]
You should be able to just set this as a default for the flag instead, but you'll have to make the version a global constant, eg. (in `main.go`):

```go
const VERSION = "0.17.1"

// ...
    Version: VERSION,
// ...
```

**Comment by robingustafsson** [comment]
@liclac Thanks. Added a version package with the constant so that it can be referenced from other parts of the code, seems to be a pattern used in other Go projects (eg. the ones from Hashicorp).

File: js/modules/k6/http/http.go [hunk 1]
```diff
@@ -129,6 +131,8 @@ func (*HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.State,
 	}
 	if userAgent := state.Options.UserAgent; userAgent.Valid {
 		req.Header.Set("User-Agent", userAgent.String)
+	} else {
+		req.Header.Set("User-Agent", fmt.Sprintf("k6/%s (https://k6.io/);", version.Version))
```
Comment (liclac) [inline js/modules/k6/http/http.go]: Please make this a default for the flag instead.

Comment (robingustafsson) [reply js/modules/k6/http/http.go]: @liclac There's no cli flag for the user agent, are you suggesting we add one?

Comment (liclac) [reply js/modules/k6/http/http.go]: Yeah. There's one in the cobra branch and I sometimes forget where the differences are...

Comment (robingustafsson) [reply js/modules/k6/http/http.go]: @liclac Ok, I'll wait until the cobra branch is merged then, and then implement this on top of that.

**Comment by liclac** [comment]
The `cobra` branch is now merged!

**Comment by borjacampina** [comment]
"ua" can be a good shorthand for user-agent, the cmd/options.go could be like this:
optionFlagSet.StringP("user-agent", "ua", fmt.Sprintf("k6/%s (https://k6.io/);", Version), "user agent for http requests")

**Comment by liclac** [comment]
Closing in favour of #406

**Pull request closed without merging**